cvmfs/CMakeLists.txt

```diff
@@ -244,6 +244,8 @@ set (CVMFS_SWISSKNIFE_SOURCES
   sync_item.cc
   sync_mediator.cc
   sync_union.cc
+  sync_union_overlayfs.cc
+  sync_union_aufs.cc
```

Comment:
These changes should be applied to the unit test CMakeLists.txt, too

Reply:
Actually I am checking it right now, but there is not even a sync_union.cc in that CMakeList, it makes sense since it is not tested at all.

I think would be more logical to add them as soon as they are needed.

Reply:
Ah, ok.  Yes, that makes sense.